modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/QuestionnaireToCsvProcessor.java

```diff
@@ -79,7 +79,7 @@ public String serialize(final Resource resource)
         // The proper serialization depends on "deep", "dereference", and "-labels", but we may allow other JSON
         // processors to be enabled/disabled to further customize the data, so we also append the original selectors
         final String processedPath = resource.getPath()
-            + resource.getResourceMetadata().getResolutionPathInfo() + ".deep.dereference.-labels";
+            + resource.getResourceMetadata().getResolutionPathInfo() + ".csv.deep.dereference.-labels";
```

Comment:
This seems unrelated to this issue.

```suggestion
            + resource.getResourceMetadata().getResolutionPathInfo() + ".deep.dereference.-labels";
```